internal/client/custom_model_service.go

```diff
@@ -57,7 +57,7 @@ type CustomModelVersion struct {
 	Dependencies             []Dependency             `json:"dependencies"`
 	RuntimeParameters        []RuntimeParameter       `json:"runtimeParameters"`
 	Items                    []FileItem               `json:"items"`
-	MaximumMemory            *int64                   `json:"maximumMemory"`
+	MaximumMemory            *float64                 `json:"maximumMemory"`
```

Comment:
Noteably not in this diff is: https://github.com/datarobot-community/terraform-provider-datarobot/pull/199/files#diff-285417b43ec40b5f9feb83c75ed265c924547e7968031d9dd458e92d153e69b2R117

which is a different struct for sending to that endpoint and is staying as an int64 to conform to the openapi spec